Jenkinsfile

```diff
@@ -16,13 +16,11 @@ pipeline {
           environment {
             CODECOV_TOKEN = credentials('cratedb-codecov-token')
           }
-          tools {
-            jdk 'jdk11'
-          }
           steps {
             sh 'git clean -xdff'
             checkout scm
-            sh 'timeout 30m ./gradlew --no-daemon --parallel -PtestForks=8 test forbiddenApisMain jacocoReport'
+            sh 'jabba install openjdk@1.11.0-2'
+            sh 'JAVA_HOME=$(jabba which --home openjdk@1.11.0-2) timeout 30m ./gradlew --no-daemon --parallel -PtestForks=8 test forbiddenApisMain jacocoReport'
```

Comment:
isn't `jabba install` already setting `JAVA_HOME`?

Reply:
No it doesn't. There would be `jabba use` which is supposed to set it, but it doesn't seem to work either.

Reply:
Ok strange, on my machine its adjusted after an `jabba install?...

Reply:
```
[admin@jenkins-09 ~]$ jabba install openjdk@1.13.0-21 && echo $JAVA_HOME

[admin@jenkins-09 ~]$ 
```